ci/prow/set_boskos_permissions.sh

```diff
@@ -17,6 +17,10 @@
 set -e
 
 readonly PROJECT=${1:?"First argument must be the boskos project name."}
+# Remaining arguments are resources to be added, just like the RESOURCES array below.
```

Comment:
It might be not able to achieve easily, but still want to see if it's possible to enforce the first argument to be project

Reply:
If it's not, gcloud in line 40 will fail and the script aborted. Ideally we would use flags to avoid confusion, but that would make this script too complicated, it's not worth it.